### ISSUE TITLE
Stacked Histogram with bars horizontal bars instead of vertical

### ISSUE THREAD

**Issue by cpotter1** [knowm/XChart#411]
Was using the examples and trying to figure out how to do that.  Not an issue, just a question - thanks!

**Comment by cpotter1** [comment]
What I am trying to do is chart start time and duration (i.e., gannt charts).  Basically looking for an example for that.

I was going to use a stacked histogram with the base "blanked out" to portray start time, and the visible section would be showing duration of the task.

Not sure if there is a better way to do it..

**Comment by Mr14huashao** [comment]
@cpotter1 @timmolter 
This graph is not supported.

**Comment by timmolter** [comment]
Yeah, sorry, it's not yet supported. i think there is already an open feature request for it.
